narwhal/consensus/src/metrics.rs

```diff
@@ -23,6 +24,21 @@ pub struct ConsensusMetrics {
     pub recovered_consensus_output: IntCounter,
     /// The approximate size in memory (including heap allocations) of the Dag.
     pub dag_size_bytes: IntGauge,
+    /// The latency between two successful commit rounds
+    pub commit_rounds_latency: Histogram,
+    /// The number of certificates committed
+    /// per commit round
+    pub commit_depth: Histogram,
```

Comment:
nit: I think `commit_depth` might be slightly confusing (I would think of it as number of rounds/leaders we committed in a single go). Perhaps `committed_certificates` is more straightforward?
 
Also I don't have super strong feelings about it, but commits usually happen roughly once a second (we have 2 rounds per second and commit can happen every 2 rounds). Do we really need a Histogram here or would simple Gauge suffice? Same question for for `commit_rounds_latency`

Reply:
Hey @andll , regarding the `commit_depth` yes, I can change this to `committed_certificates` . Regarding its role, although we do commit every 2 rounds, we don't have guarantees on the amount of certificates committed (ex because we missed some leaders). I would say its value will come up mostly in the edge cases

For the `commit_rounds_latency` I use a Histogram to make sure that all the sampling gets captured. The way that prometheus works (polling the metrics instant value every 30 seconds) we'll probably just see whatever the latest latency was at that point - whereas I would like to see all the observations (so get the p95, p90 etc)